samples/dnn/models.yml

```diff
@@ -15,11 +15,11 @@ opencv_fd:
   rgb: false
   sample: "object_detection"
 
-# YOLO object detection family from Darknet (https://pjreddie.com/darknet/yolo/)
-# Might be used for all YOLOv2, TinyYolov2 and YOLOv3
+# YOLO object detection family from Darknet (https://github.com/AlexeyAB/darknet#pre-trained-models)
```

Comment:
Please refer to origin repository as well:
```
# YOLO object detection family from Darknet. Might be used for
#   * YOLOv2, TinyYolov2 and YOLOv3 (https://pjreddie.com/darknet/yolo/)
#   * YOLOv4 (https://github.com/AlexeyAB/darknet)
```

Reply:
ok